src/Sentry.GraphQl/SentryGraphQlHttpFailedRequestHandler.cs

```diff
@@ -0,0 +1,93 @@
+namespace Sentry.GraphQl;
+
+internal class SentryGraphQlHttpFailedRequestHandler : SentryFailedRequestHandler
+{
+    private readonly IHub _hub;
+    private readonly SentryOptions _options;
+    private readonly GraphQlContentExtractor _extractor;
+    internal const string MechanismType = "SentryGraphQLHttpFailedRequestHandler";
+    private readonly SentryHttpFailedRequestHandler _httpFailedRequestHandler;
+
+    internal SentryGraphQlHttpFailedRequestHandler(IHub hub, SentryOptions options, GraphQlContentExtractor? extractor = null)
+        : base(hub, options)
+    {
+        _hub = hub;
+        _options = options;
+        _extractor = extractor ?? new GraphQlContentExtractor(options);
+        _httpFailedRequestHandler = new SentryHttpFailedRequestHandler(hub, options);
+    }
+
+    private static readonly Regex ErrorsRegex = new ("(?i)\"errors\"\\s*:\\s*\\[", RegexOptions.Compiled);
```

Comment:
Should that get used somewhere?

Reply:
Ah no... that was from the requirements but we're using the functionality on JsonElement to see if there is an `errors` property present (seemed cleaner to me). I'll get rid of the regex.